ArticleTemplates/assets/scss/base/_typography.scss

```diff
@@ -145,35 +145,29 @@ $fontMap: (
     // LEGACY FONTS:
 
     // Display Egyptian
-    GuardianEgyptianWeb100: (
-        family: 'Guardian Egyptian Web',
-        weight: 100,
-        android-src: 'url("../fonts/egyptiandisplay-thin.otf")',
-        ios-src: 'url("../fonts/egyptiandisplay-thin.otf")'
-    ),
     GuardianEgyptianWeb200: (
         family: 'Guardian Egyptian Web',
         weight: 200,
-        android-src: 'url("../fonts/egyptiandisplay-light.otf")',
-        ios-src: 'url("../fonts/egyptiandisplay-light.otf")'
+        android-src: 'url("../fonts/GHGuardianHeadline-Light.ttf")',
+        ios-src: 'url("../fonts/GHGuardianHeadline-Light.ttf")'
     ),
     GuardianEgyptianWeb300: (
         family: 'Guardian Egyptian Web',
         weight: 300,
-        android-src: 'url("../fonts/egyptiandisplay-regular.otf")',
-        ios-src: 'url("../fonts/egyptiandisplay-regular.otf")'
+        android-src: 'url("../fonts/GHGuardianHeadline-Regular.ttf")',
+        ios-src: 'url("../fonts/GHGuardianHeadline-Regular.ttf")'
     ),
     GuardianEgyptianWeb400: (
         family: 'Guardian Egyptian Web',
         weight: 400,
-        android-src: 'url("../fonts/egyptiandisplay-medium.otf")',
-        ios-src: 'url("../fonts/egyptiandisplay-medium.otf")'
+        android-src: 'url("../fonts/GHGuardianHeadline-Medium.ttf")',
+        ios-src: 'url("../fonts/GHGuardianHeadline-Medium.ttf")'
     ),
     GuardianEgyptianWeb600: (
         family: 'Guardian Egyptian Web',
         weight: 600,
-        android-src: 'url("../fonts/egyptiandisplay-bold.otf")',
-        ios-src: 'url("../fonts/egyptiandisplay-bold.otf")'
+        android-src: 'url("../fonts/GHGuardianHeadline-Bold.ttf")',
+        ios-src: 'url("../fonts/GHGuardianHeadline-Bold.ttf")'
```

Comment:
Are these weights correct? They're all 100 off the [common name mappings](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Common_weight_name_mapping):

- Light: 300
- Regular: 400
- Medium: 500
- Bold: 700

cc @paperboyo

Reply:
Common are correct and used in other places in the Guardian. Looks like here they have just been taken out of a hat 😄.
No idea what making them correct here would mean, though?

Reply:
@duarte any thoughts?

Reply:
I have nothing to add Re: weight numbers

Reply:
Having rebased the changes in this file go away. I think they were already made in #1163 and not being picked up correctly.